### PR TITLE
[issue/10] - Added report setup

### LINKED ISSUE #10: Setup report structure in report repository

### PR DESCRIPTION
**Pull request by carina-str**
- Added markdown files
- Added folder structure
- Added image folder for all charts / diagrams and images

closes #10

### CODE REVIEW

File: report/00_abstract/00_abstract.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Abstract
+
+Probably everyone knows the feeling of fatigue. It is a natural signal of the human body to show that it needs a break and rest. However, as soon as this state of fatigue lasts longer and does not go back through short breaks, you should think about cognitive fatigue. This is a state of fatigue, that persists after long-term cognitive and mental stress. This report now looks at a methodology to use a cross platform application to determine cognitive fatigue.
```
Comment (mac641) [inline report/00_abstract/00_abstract.md:3]:
```suggestion
Being fatigued is a common state of everyone's daily life. It is a natural sign of your body that signals you to have a break and rest. However, as soon as this state of fatigue lasts longer and does not go back through short breaks, you could consider yourself being cognitively fatigued. This type of fatigue persists after long-term cognitive and mental stress. The purpose of this report is to look at a methodology to use a cross platform application to determine cognitive fatigue.

```

File: report/01_introduction/00_introduction.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+The application should therefore be able to determine the general condition of a subject with the help of a questionnaire that should be filled out regularly and replace earlier paper questionnaires and be able to save it centrally in a database. In addition, a self-test should be available, which in the best case should be completed several times a day. This should contain two tasks that check the respondent's reaction time and memory ability. In addition, with each run of the self-test, the current emotional state and the current activity of the test person are recorded so that conclusions can be drawn from them later. This information is recorded in a database that is to be exchanged regularly with a server database in order to be able to export all user data centrally for a study.
+
+
+[1] Shortz, A.E., Pickens, A., Zheng, Q. et al. The effect of cognitive fatigue on prefrontal cortex correlates of neuromuscular fatigue in older women. J NeuroEngineering Rehabil 12, 115 (2015). https://doi-org.ezproxy.hs-augsburg.de/10.1186/s12984-015-0108-3
```
Comment (mac641) [inline report/01_introduction/00_introduction.md:11]: I'd put sources in a separate file at the end of the report 🤔

**Review by mac641 — CHANGES_REQUESTED** [review]
Project setup looks good to me - I like 😄

I've commented some suggestions that came to mind while reading ;)
If you disagree, let me know and we can discuss them :)

File: report/01_introduction/00_introduction.md (hunk 2 above)
Comment (carina-str) [reply report/01_introduction/00_introduction.md:11]: Yes, the seperate file is already there, but in order not to get confused, it's hard to find unambiguous names for the references. I think when we're working on it at the same time, it's difficult to set it in a global place. My suggestion would be to first put the references in the respective file and at the end someone looks over it and summarizes it and sets the numbering?

**Review by mac641 — APPROVED** [review]
🎉

**Review by diehmlukas — APPROVED** [review]
I like it! :1st_place_medal:

**Pull request merged**